src/Bridge/Twig/Extension/SocialImageExtension.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sigwin Yassg project.
+ *
+ * (c) sigwin.hr
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sigwin\YASSG\Bridge\Twig\Extension;
+
+use Sigwin\YASSG\Asset\AssetFetch;
+use Sigwin\YASSG\AssetQueue;
+use Symfony\Component\Asset\Packages;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Twig\Environment;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+final class SocialImageExtension extends AbstractExtension
+{
+    public function __construct(
+        private readonly RequestStack $requestStack,
+        private readonly string $imgproxyUrl,
+        private readonly Packages $packages,
+        private readonly AssetQueue $thumbnailQueue,
+        private readonly Environment $twig,
+        private readonly Filesystem $filesystem,
+        private readonly string $buildDir,
+        private readonly string $baseDir,
+        private readonly ?string $defaultSocialImageTemplate,
+        private readonly array $routes
+    ) {
+    }
+
+    #[\Override]
+    public function getFunctions(): array
+    {
+        return [
+            new TwigFunction(
+                'yassg_social_image',
+                /**
+                 * @param array<string, mixed> $context
+                 * @param array<string, mixed> $options
+                 */
+                function (array $context, ?string $template = null, array $options = []): string {
+                    return $this->generateSocialImage($context, $template, $options);
+                },
+                ['needs_context' => true]
+            ),
+        ];
+    }
+
+    /**
+     * @param array<string, mixed> $context
+     * @param array<string, mixed> $options
+     */
+    private function generateSocialImage(array $context, ?string $template, array $options): string
+    {
+        // Determine which template to use
+        $templatePath = $this->resolveTemplate($template, $context, $options);
+        
+        if ($templatePath === null) {
+            throw new \RuntimeException('No social image template configured. Set sigwin_yassg.social_image_template or pass a template path.');
+        }
+
+        // Render the SVG template with the current context
+        $svgContent = $this->twig->render($templatePath, $context);
+
+        // Create a hash for the SVG content (using SHA-256 for collision resistance)
+        $hash = hash('sha256', $svgContent);
```

Comment:
I don't think we need any of this because we don't actually want to build the SVG, we want the WebP/AVIF which gets dumped from it.

The idea is something like

```
{{ yassg_thumbnail(yassg_social_image(article)) }}
```

Which would basically produce a link to the SVG passed through ImgProxy. This would then also work when dumping by default because it's just another image.